src/Samples/PsdTgaExporter.cpp

```diff
@@ -16,6 +16,13 @@ void OutputDebugStringA(const char *message)
 }
 #endif
 
+#ifdef __linux
+#include <cwchar>
+#include <cstring>
+#include <cstdlib>
+#define OutputDebugStringA(S) fputs(S,stderr)

```

Comment:
I think it would be better to also put this into a separate file.
Maybe get rid of OutputDebugString altogether and change it into something platform-agnostic, which then calls OutputDebugStringA on Windows, and fputs on Linux?

Reply:
I had put them into psdDebug.h